cfme/tests/networks/nuage/test_events_are_triggered.py

```diff
@@ -95,6 +151,38 @@ def test_creating_entities_emits_events(register_event, with_nuage_sandbox):
     expect_event(listener, 'nuage_policygroup_create', sandbox['l2_group'].id, comment='L2')
 
 
+def test_targeted_refresh_after_create_entities(targeted_refresh, with_nuage_sandbox):
+    """
+    Test whether targeted refresh is triggered when entities are created. This way
+    we check whether Automation Instances are properly set to trigger targeted refresh
+    when event comes.
+
+    config:
+        Before running this test you have to configure merkyl (see how in
+        cfme/fixtures/merkyl.py) and make sure you have port 8192 opened
+        on your appliance.
+
+    Steps:
+        * Deploy entities outside of CFME (directly in the provider)
+        * Verify that targeted refreshes where triggered
+
+    Important: targeted_refresh fixture must be listed before with_nuage_sandbox
+    fixture because we have to start tracking evm.log before we create the entites
+    """
+    sandbox = with_nuage_sandbox
+    with targeted_refresh.timeout():
+        targeted_refresh.register_target(sandbox['enterprise'].id, 'Enterprise')
+        targeted_refresh.register_target(sandbox['domain'].id, 'Domain')
+        targeted_refresh.register_target(sandbox['subnet'].id, 'Subnet')
+        targeted_refresh.register_target(sandbox['cont_vport'].id, 'Container vPort (L3)')
+        targeted_refresh.register_target(sandbox['vm_vport'].id, 'VM vPort (L3)')
+        targeted_refresh.register_target(sandbox['l2_domain'].id, 'Domain (L2)')
+        targeted_refresh.register_target(sandbox['l2_cont_vport'].id, 'Container vPort (L2)')
+        targeted_refresh.register_target(sandbox['l2_vm_vport'].id, 'VM vPort (L2)')
+        targeted_refresh.register_target(sandbox['group'].id, 'L3')
+        targeted_refresh.register_target(sandbox['l2_group'].id, 'L2')
```

Comment:
Fix the last tow comments, please:
'L3' -> 'Security group (L3)'
'L2' -> 'Security group (L2)'